### ISSUE TITLE
How do you grab other paraments via xpath other then identifiers?

### ISSUE THREAD

**Issue by odonnellja** [FlaUI/FlaUI#517]
I am trying to assert on values in certain boxes using the FlaUI library with Robot Framework. Can someone help me with the xpath identifier that I should use. 
My xpath to find the specific element is:
 '/Window[@Name='RigSense Main Client']/Window[@Name='Setup Charts']/Pane[1]/Pane[1]/Pane[3]/Edit[@AutomationId='txtPensPerChart']'

The screenshot shows the FlaUInspect for that element, I want assert that the "Value" parameter is '1'. So how do I either save that value parameter as a variable and then i can 'should be equal' to it or is there a better way?

![image](https://user-images.githubusercontent.com/78232961/193669578-3771ca31-60f2-4ff1-a478-6cb5a1808b1c.png)

**Comment by DNetORM** [comment]
Yes, I also want to know how to create xpath expressions. Where can I see the tutorial about xpath
